services/github/github-common-release.js

```diff
@@ -68,6 +76,8 @@ function getLatestRelease({ releases, sort, includePrereleases }) {
 const queryParamSchema = Joi.object({
   include_prereleases: Joi.equal(''),
   sort: Joi.string().valid('date', 'semver').default('date'),
+  prefix: Joi.any(),
+  subpackage: Joi.any(),
```

Comment:
Once we've sorted out what the params are, we shouldn't use `Joi.any()`. They need a type. Probably should be `Joi.string().required()` but if we actually need to accept multiple types, we can use https://joi.dev/api/?v=17.6.0#alternatives

Reply:
I used `Joi.any()` over `Joi.string()` because when I had something that it thought was a number it would return undefined. So a prefix of `0.1` Showed up as undefined. I will look into accepting multiple.

Reply:
Fixed. Thanks!